src/screens/RegisterScreen/LoginScreen.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import S from './LoginScreen.style';
+import {signInWithKakao, signInWithNaver} from '@/apis/Login';
+import {useNavigation} from '@react-navigation/native';
+import {StackNavigationProp} from '@react-navigation/stack';
+import {RootStackParamList} from '@/types/StackNavigationType';
+
+const LoginScreen = () => {
+  const navigation = useNavigation<StackNavigationProp<RootStackParamList>>();
+
+  const handleKakaoLogin = async () => {
+    await signInWithKakao(navigation);
+  };
+
+  const handleNaverLogin = async () => {
+    await signInWithNaver(navigation);
+  };
```

Comment:
소비자 앱과 마찬기지로 비동기에 대한 에러핸들링이 추가되면 좋겠습니다. TODO로 남겨두죠.
```suggestion
  // TODO: 로그인 에러 핸들링 추가하기
  const handleKakaoLogin = async () => {
    await signInWithKakao(navigation);
  };

  const handleNaverLogin = async () => {
    await signInWithNaver(navigation);
  };
```

Reply:
코드 고치다가 삭제되었나보네요. 수정해서 커밋올리고 머지 해놓겠습니다.